src/inputs/Multi.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div class="form__radios">
+        <span>{{ this.label }}</span>
+        <div v-for="item in inputValue" :key="item.key"> 
+            <component
+                :value="item.value"
+                :identifier="item.key"
+                :isMulti=true
+                :is="itemType"
+                v-bind="item.props"
+                :index="item.index"
+                v-on:onChange="change"
+                :name="item.name" />
+            <button @click="e => removeInput(e, item.key)"/>
+        </div>
+        <input type="submit" value="Add New" v-on:click="e => addNew(e)">
+        <field-error :errors="this.validationErrors" />
+    </div>
+</template>
+
+<script>
+    import Vue from 'vue';
+    import { SuperText } from '../';
+    import { validationMixin } from '../validationHelper';
+    import { formSchemaMixin } from '../schemaHelper';
+    import _ from 'lodash';
+
+    export default {
+        name: 'SuperMulti',
+        mixins: [validationMixin, formSchemaMixin],
+        mounted() {            
+            this.inputValue = this.inputValue ? this.inputValue.map((val, i) => (
+                this.itemObjectGenerator(val, i)
```

Comment:
you're returning a single value, so brackets are not needed here. You can do this:
```javascript
this.inputValue.map((val, i) => this.itemObjectGenerator(val, i))
```